lib/systems/inspect.nix

```diff
@@ -6,6 +6,8 @@ with lib.lists;
 let abis_ = abis; in
 let abis = lib.mapAttrs (_: abi: builtins.removeAttrs abi [ "assertions" ]) abis_; in
 
+# TODO: isUefi
+
```

Comment:
Can you write at least a half sentence otherwise we forget what we wanted to do here

Reply:
It was supposed to add a `isUefi` function here so we can test for it